src/app/components/card-entreprise/card-entreprise.component.html

```diff
@@ -0,0 +1,23 @@
+<!--- Cartes de entreprises --->
+
+<a *ngFor="let entreprise of entreprises" routerLink="">
+    <mat-card class="entrepriseCard">
+        <img src="../../../assets/img/logoEntreprise.jpg" alt="Logo entreprise">
```

Comment:
Il y a un champ pour l'URL dans l'entreprise: `entreprise.imageUrl`. Le formulaire d'ajout permet déjà d'en ajouter un. 
```suggestion
        <img [src]="entreprise.imageUrl || '../../../assets/img/no-logo.jpg" alt="Logo entreprise">
```
Petite suggestion qui permet d'afficher un logo générique si `imageUrl` est une string vide.

Reply:
ok je modifie